### PR TITLE
fix bug to add authority and show read recoders

### PR DESCRIPTION
**Pull request by JiuXinnuomi**
fix bug #4

### CODE REVIEW

**Comment by JiuXinnuomi** [comment]
the first pull requests

**Comment by JiuXinnuomi** [comment]
the first pull requests for me

**Comment by graydovee** [comment]
fix bug #4

**Review by graydovee — COMMENTED** [review]
format code

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created by .ignore support plugin (hsz.mobi)
+.gitignore
+.idea/.gitignore
```
Comment (graydovee) [inline .gitignore:3]: instead of ".idea"

**Pull request merged**